etl/transform/generate_datadiscovery.py

```diff
@@ -2,8 +2,16 @@
 from etl.transform.utils import get_generated_uri_from_str
 
 
+def make_hashable(obj):
```

Comment:
Add a comment explaining the origin of this method: synonyms were either list(string) or list(obj) which generate hash error. This methods solv the problem but the real problem was the data model that had changed.